.changeset/better-parks-warn.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/server': minor
+---
+
+Added advanced trace-query support for richer same-span filters while keeping lightweight responses unchanged.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)